internal/app/skuba/cluster.go

```diff
@@ -34,6 +34,7 @@ func NewClusterCmd() *cobra.Command {
 		cluster.NewInitCmd(),
 		cluster.NewStatusCmd(),
 		cluster.NewUpgradeCmd(),
+		cluster.NewImagesCmd(),
```

Comment:
We should document this command here: https://github.com/SUSE/skuba/tree/master/docs/man, creating its own documentation man page, and also changing https://github.com/SUSE/skuba/blob/master/docs/man/skuba.1.md `See Also` section.

Reply:
I have created a man page and updated skuba.1.md.